var/spack/repos/builtin/packages/libmesh/package.py

```diff
@@ -180,7 +180,7 @@ def configure_args(self):
 
         # and, finally, other things:
         if '+debug' in self.spec:
-            options.append('--with-methods=dbg')
+            options.append('--with-methods="dbg opt"')
```

Comment:
I think this would be a great solution! It fits well with how it is already done in the libMesh install procedure as you said. Stupid question though (I'm still kind of new with Spack), but how do you specify multiple options on the command line? Would it just be `build_type="debug release"` or as something else?

Reply:
Well..after doing a quick test, I see that you specify multiple value variants with `build_type="debug, release"`.